docs/community/tutorials-contest/2022/red-dev-ecrecover-x-chain/README.md

```diff
@@ -0,0 +1,368 @@
+# How to Extend Avalanche's Subnet-evm with a stateful Precompile
```

Comment:
All section header should be title case, please check and update all section headers with https://titlecase.com/